build/docs/build_docs.py

```diff
@@ -138,10 +140,10 @@ def format_as_document(magic, uid_base : str) -> MagicReferenceDocument:
     remarks = format_as_section('Remarks', doc.get('Remarks', ""))
 
     raw_examples = doc.get('Examples', [])
-    examples = "\n".join(
-        format_as_section("Example", example)
-        for example in raw_examples
-    ) if raw_examples else ""
+    examples = format_as_section('Examples', "\n".join(
```

Comment:
For search purposes, can we clarify the examples section - (f"Examples for {magic_name}")

Reply:
Yes, that's a good idea!